bin/coverage-checker.php

```diff
@@ -5,16 +5,16 @@
 declare(strict_types=1);
 
 $inputFile = $argv[1];
+
+if (!is_numeric($argv[2])) {
+    throw new InvalidArgumentException('An integer checked percentage must be given as second parameter');
+}
 $percentage = min(100, max(0, (int) $argv[2]));
 
 if (!file_exists($inputFile)) {
     throw new InvalidArgumentException('Invalid input file provided');
 }
 
-if (!is_int($percentage)) {
```

Comment:
Detected by PHPStan bleeding edge:
```
------ ----------------------------------------------------------------------------------------------------------------------
  Line   bin/coverage-checker.php
 ------ ----------------------------------------------------------------------------------------------------------------------
  14     Call to function is_int() with int<0, 100> will always evaluate to true.
         🪪  function.alreadyNarrowedType
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false
            in your phpstan.neon.
 ------ ----------------------------------------------------------------------------------------------------------------------
```